aten/src/ATen/native/cpu/FusedAdamKernel.cpp

```diff
@@ -238,7 +238,7 @@ std::enable_if_t<
         param_ptr[d] = param_ptr[d] * scalar_t(1 - lr * weight_decay);
       }
     }
-    // exp_avg.lerp_(grad, 1 - beta1) if beta1 > 0 else exp_avg = grad 
+    // exp_avg.lerp_(grad, 1 - beta1) if beta1 > 0 else grad  
```

Comment:
why the comment change? what you had before was clearer! 
```suggestion
    // exp_avg.lerp_(grad, 1 - beta1) if beta1 > 0 else exp_avg = grad  
```